support/testing/cid.go

```diff
@@ -0,0 +1,22 @@
+package testing
+
+import (
+	"github.com/ipfs/go-cid"
+	cbor "github.com/ipfs/go-ipld-cbor"
```

Comment:
We don't really want this dep in the repo at all. It's unfortunate that this package brings the IPLD store interface (good) along with a deep dependency on refmt and etc (bad). Please don't use WrapObject.

I much prefer explicit construction to this like this stateful "getter" pattern. Use something like:
```
var DefaultCidBuilder = cid.V1Builder{Codec: cid.DagCBOR, MhType: DefaultHashFunction}
// Or cid.V1Builder{Codec: cid.Raw, MhType: mh.IDENTITY} for more transparent CIDs
func CidFromString(t *testing.T, input string) cid.Cid {
	c, err := constants.DefaultCidBuilder.Sum([]byte(input))
	require.NoError(t, err)
	return c
```
And give the CIDs explicit values.